timescaledb/how-to-guides/toolkit/counter_agg.md

```diff
@@ -0,0 +1,164 @@
+# Counter aggregates
+There are two types of metrics that can be recorded in a database: gauges, and
+counters. Gauges fluctuate up and down, like temperature or speed, while
+counters always increase, like the total number of visitors to a website.
+
+When you process counter data, it is usually assumed that if the value of the
+counter goes down, the counter has been reset. For example, if you wanted to
+count how many times a runner ran around a track, you would expect the values to
+continuously increase: 1, 2, 3, 4, and so on. If the counter reset to 0, you
+would expect that this was a new runner starting on their first lap, not the
+original runner going back to 1. This can become a problem if you want to
+continue counting from where you left off, rather than resetting to 0. A reset
+could occur if you have had a short server outage, or any number of other
+reasons. To get around this, you can analyze counter data by looking at the
+change over time, which accounts for resets.
+
+Accounting for resets can be difficult to do in SQL, so Timescale has developed
+aggregate and accessor functions that handle calculations for counters in a more
+practical way.
+
+<highlight type="note">
+Counter aggregates can be used in continuous aggregates, even though they are
+not parallelizable in PostgreSQL. For more information, see the section on
+[parallelism and ordering][#parallelism-and-ordering].
+</highlight>
+
+## Run a counter aggregate query
+In this procedure, we are using an example table called `example` that contains
+counter data.
+
+### Procedure: Running a counter aggregate query
```

Comment:
I still, really really hate calling these procedures, can we call it an example? The overloading with SQL procedures is really, really confusing to me as a SQL user...